src/relay/backend/te_compiler_cache.cc

```diff
@@ -45,6 +45,8 @@
 #include "../../te/operation/create_primfunc.h"
 #include "../op/memory/memory.h"
 #include "../transforms/pass_utils.h"
+#include "tvm/relay/op_strategy.h"
+#include "tvm/tir/function.h"
```

Comment:
nit: Maybe we can follow the convention using

```suggestion
#include <tvm/relay/op_strategy.h>
#include <tvm/tir/function.h>
```

Reply:
They were auto-inserted by `clangd` :man_facepalming: